docs/tutorials/blog.md

```diff
@@ -1037,8 +1037,8 @@ We hope you love Remix! 💿 👋
 [the-styling-guide]: /guides/styling
 [prisma]: https://prisma.io
 [http-localhost-3000-posts-admin]: http://localhost:3000/posts/admin
-[mdn-io-request]: https://mdn.io/Request
-[mdn-io-request-form-data]: https://mdn.io/Request.formData
+[mdn-io-request]: https://developer.mozilla.org/en-US/docs/Web/API/Request
+[mdn-io-request-form-data]: https://developer.mozilla.org/en-US/docs/Web/API/Request/formData
```

Comment:
Please remove `-io` from both labels and update the references above 🙏 will be good to merge after that

Reply:
```suggestion
[mdn-request]: https://developer.mozilla.org/en-US/docs/Web/API/Request
[mdn-request-form-data]: https://developer.mozilla.org/en-US/docs/Web/API/Request/formData
```